src/components/form-helpers/form-helpers.js

```diff
@@ -173,6 +213,40 @@ export const PasswordFieldItem = ({ className, fieldName, label, placeholder, he
   );
 };
 
+export const PasswordValidatedFieldItem = ({ className, fieldName, label, placeholder }) => {
```

Comment:
What do you think about `ValidatedPasswordFieldItem` or `PasswordDefinitionFieldItem`?

Reply:
If I am not wrong, a big part of _PasswordValidatedFieldItem_ content and _PasswordFieldItem_ is the same, you could move it into a private component and reuse it in both components.

Reply:
This one `ValidatedPasswordFieldItem` it's better. I like the word validated because the difference between this field and the one in the login is the validation. `Definition` I think would not fit here. My order of words is a bit weird so I will change it. Thanks!